Directory.Packages.props

```diff
@@ -120,6 +120,7 @@
     <PackageVersion Include="StackExchange.Redis" Version="2.7.33" />
     <PackageVersion Include="System.IO.Hashing" Version="8.0.0" />
     <PackageVersion Include="Yarp.ReverseProxy" Version="2.1.0" />
+    <PackageVersion Include="Elasticsearch.Net" Version="7.17.5" />
```

Comment:
Deprecated package?

![image](https://github.com/dotnet/aspire/assets/513398/e3a4d615-be6f-4765-95ed-3da9a04f2b7c)

Reply:
I have decided to add a client component with the `Elastic.Clients.Elasticsearch` package soon.
I used this deprecated package because `Elastic.Clients.Elasticsearch` was not available in the mirror package source until now.